core/theme/ThemeManager.php

```diff
@@ -53,6 +53,7 @@ protected static function loadThemeConfig(string $basePath)
         if (strpos($basePath, '@') === 0) {
             $dir = Yii::getAlias($basePath);
         } elseif (strpos($basePath, '/') !== 0) {
+            // todo: keep basepath as relative path
```

Comment:
This brings me into a bit complicated situation 😄 

We should remove this comment or either fix it. You know what i mean? This won't help anyone reading our code, it even might bring up some questions.

Reply:
Of course I understand. I will create an issue for this, because it have to be testet good.

Reply:
New Issue https://github.com/luyadev/luya/issues/1970